Makefile

```diff
@@ -34,7 +34,7 @@ check-release:
 	@echo Released srclib $(V)
 
 install-std-toolchains:
```

Comment:
@rothfels this is just part of the Makefile, so it won't end up as part of the srclib command. There is also `test-std-toolchains`, which I suspect we need. I'll rename them to something more appropriate though.